op-chain-ops/cmd/celo-migrate/ancients.go

```diff
@@ -144,6 +144,9 @@ func readAncientBlocks(ctx context.Context, freezer *rawdb.Freezer, startBlock,
 func transformBlocks(ctx context.Context, in <-chan RLPBlockRange, out chan<- RLPBlockRange) error {
 	// Transform blocks from the in channel and send them to the out channel
 	defer close(out)
+
+	prevBlockNumber := uint64(0)
```

Comment:
This relies on the assumption that the incoming block ranges are ordered, which is not obvious. Maybe add a comment?